src/Checkbox/style.css

```diff
@@ -8,7 +8,7 @@
   }
 
   &-wrapper {
-    @apply text-slate-700 font-inter font-normal flex;
+    @apply text-slate-700 font-inter font-normal;
```

Comment:
Did you make sure this didn't break anything?

Reply:
@AladdinNullptr yes i checked the `Checkbox` component and other components that use `Checkbox`. nothing breaks.